app/views/pages/about.html.slim

```diff
@@ -16,13 +18,14 @@
   p = t('layout.info.about.p3')
   p = t('layout.info.about.p4')
   h2 = t('layout.info.about.p6')
-  ul
-    li = t('layout.info.about.p7')
-    li = t('layout.info.about.p8')
-    li = t('layout.info.about.p9')
-    li = t('layout.info.about.p10')
+  ul 
+    - t('layout.info.about.do_list').each do |item|
```

Comment:
cool! I've not used lists in I18n before, but I think this is a good time to do that